packages/picasso.js/src/core/chart/index.js

```diff
@@ -256,6 +256,7 @@ function chartFn(definition, context) {
     hidden.forEach((comp) => {
       comp.instance.hide();
       comp.visible = false;
+      comp.instance.setVisibility(comp.visible);
```

Comment:
there is no need to make any changes in this file to implement this, use `mount/unmount` in `component-factory.js` to keep track of the visibility of the component